### PR TITLE
DLS-9282 Content changes - page title update

### PR DESCRIPTION
**Pull request by karolina-zaborska**
- changed the page title content for H1 for both Welsh and English
- removed the hint section for accessibility, replaced it with a paragraph and fixed the failing tests associated with that change
- implemented 'label' label and all relevant keys in both messages files
- rearranged the structure of HTML to address the requested changes
- code reformat

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/disposal_price.scala.html [hunk 1]
```diff
@@ -73,20 +74,27 @@
...
+    @details(
+        messages(s"$key.link"),
+        Html(
+            s"""
+          | <p class="govuk-body">${messages(s"$key.details.p1")}</p>
+          |""".stripMargin
+        )
+    )
+
+    <h2 class="govuk-fieldset__legend govuk-fieldset__legend--m">@h2</h2>
```
Comment (johno1985) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/disposal_price.scala.html:90]: I checked with Adam Liptrot, we do not want to be using a h2 for the input label

**Review by johno1985 — CHANGES_REQUESTED** [review]
remove h2, use regular form input label

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/disposal_price.scala.html [hunk 2]
```diff
@@ -73,40 +84,40 @@
...
           | <p class="govuk-body">${messages(s"$key.details.p1")}</p>
           |""".stripMargin
         )
-      )
+    )
+
+    @formWrapper(address.routes.PropertyDetailsController.disposalPriceSubmit()) {
+        @govukInput(Input(
+            label = Label(
+                content = Text(input),
```
Comment (johno1985) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/disposal_price.scala.html:103]: please can we call this 'label'?

File: conf/messages [hunk 3]
```diff
@@ -4518,6 +4518,14 @@ multipleDisposalsDisposalPrice.personalRepInPeriodOfAdmin.agent.helpText=This is
 multipleDisposalsDisposalPrice.link=What are disposal costs?
 multipleDisposalsDisposalPrice.details.p1=Disposal or selling costs include fees for estate agents or solicitors.
 
+multipleDisposalsDisposalPrice.input=What was the disposal value of the property?
```
Comment (johno1985) [inline conf/messages:4521]: could probably collapse this into a single message entry

**Review by johno1985 — CHANGES_REQUESTED** [review]
couple more things

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/disposal_price.scala.html (hunk 2 above)
Comment (karolina-zaborska) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/disposal_price.scala.html:103]: Just saw your discussion with Lukasz about the 'label'. You still want me to change this from 'input' to 'label'?

Comment (johno1985) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/disposal_price.scala.html:103]: Just the variable name 'input' -> 'label'

File: conf/messages (hunk 3 above)
Comment (karolina-zaborska) [reply conf/messages:4521]: I thought so too but that's the way its done in the message file so I adjusted to that to maintain consistency

**Comment by karolina-zaborska** [comment]
Wrong branch name so I've closed this PR. The correct one is below:
https://github.com/hmrc/cgt-property-disposals-frontend/pull/1269

**Pull request closed without merging**